examples/s2s-api/oxxo_payment.php

```diff
@@ -90,17 +91,24 @@
     $response = $payment->postRequest();
 
     echo "\n\nRESULT BELOW\n";
-    if ($response->getStatus() == \VendoSdk\Vendo::S2S_STATUS_VERIFICATION_REQUIRED) {
+    if ($response->getStatus() == \VendoSdk\Vendo::S2S_STATUS_OK) {
         echo "The transactions was successfully processed. Vendo's Transaction ID is: " . $response->getTransactionDetails()->getId();
-        echo "\nThe OXXO PDF (gzipped and base64 encoded): " . $response->getOxxoPaymentResult()->getPdfGzipBase64();
-        echo "\nThe OXXO PNG (gzipped and base64 encoded): " . $response->getOxxoPaymentResult()->getPngGzipBase64();
-        echo "\nThe expiration date (gzipped and base64 encoded): " . $response->getOxxoPaymentResult()->getExpiresAt();
-        echo "\nThe OXXO barcode: " . $response->getOxxoPaymentResult()->getBarcode();
+        echo "\nThe payment Auth Code is: " . $response->getCreditCardPaymentResult()->getAuthCode();
+        echo "\nThe Payment Details Token is: ". $response->getPaymentToken();
+        echo "\nYou must save the payment details token if you need or want to process one-clicks\n";
         echo "\nThis is your transaction reference (the one you set it in the request): " . $response->getExternalReferences()->getTransactionReference();
     } elseif ($response->getStatus() == \VendoSdk\Vendo::S2S_STATUS_NOT_OK) {
         echo "The transaction failed.";
         echo "\nError message: " . $response->getErrorMessage();
         echo "\nError code: " . $response->getErrorCode();
+
+    } elseif ($response->getStatus() == \VendoSdk\Vendo::S2S_STATUS_VERIFICATION_REQUIRED) {
+        echo "The transaction must be verified";
+        echo "\nYou MUST :";
+        echo "\n   1. Save the payment token: " . $response->getPaymentToken();
+        echo "\n   2. Redirect the user to the verification URL: " . $response->getResultDetails()->getVerificationUrl();
+        echo "\nthe user will verify his payment details, then he will be redirected to the Success URL that's configured in your account at Vendo's back office.";
+        echo "\nwhen the user comes back you need to post the request to vendo again, you can use the TokenPayment class.";
```

Comment:
Also, update this URL, we don't want merchants to post the token anymore, we want them to use the verification_id instead.